libp2p/src/tutorials/ping.rs

```diff
@@ -75,17 +76,14 @@
 //!
 //! #[async_std::main]
 //! async fn main() -> Result<(), Box<dyn Error>> {
+//!     env_logger::init();
 //!     let local_key = identity::Keypair::generate_ed25519();
 //!     let local_peer_id = PeerId::from(local_key.public());
-//!     println!("Local peer id: {local_peer_id:?}");
 //!
 //!     Ok(())
 //! }
 //! ```
 //!
-//! Go ahead and build and run the above code with: `cargo run`. A unique
-//! [`PeerId`](crate::PeerId) should be displayed.
```

Comment:
This is a bit odd now, isn't it? Why don't we retain the `println` here until we initalize the `Swarm` and explain in the tutorial that this is being printed implicitly? (And also only then add the logger?). I am open to other suggestions but we should be consistent / make sense here :)

Reply:
Yes, I agree it looks weird. I decided to return all `println` before the creation of `Swarm,` also added documentation and explained why we remove `println`

Reply:
Awesome, thanks!